### PR TITLE
Hover after next

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

**Comment by vitsalis** [comment]
Depends on #218

**Comment by vitsalis** [comment]
Fixes #207

File: static/script/game/view.js [hunk 1]
```diff
@@ -83,8 +84,9 @@ $( document ).ready( function() {
                         left: creature.x * 20 + 'px',
                         top: creature.y * 20 + 'px'
                     } );
+                    $creature.mouseover( creatureMouseover );
```
Comment (dionyziz) [inline static/script/game/view.js]: Use delegated events with jQuery `on()`

File: static/script/game/view.js [hunk 2]
```diff
@@ -67,10 +67,10 @@ $( document ).ready( function() {
                 var creature = creatures[ i ];
                 if ( creature.hp > 0 ) {
                     var $user = $( '.playerList li[data-id=' + creature.userid + ']' );
-                    var username = $user.text();
+                    var username = $user.text().trim();
```
Comment (dionyziz) [inline static/script/game/view.js]: Why is this needed?
